src/pages/get-involved.tsx

```diff
@@ -1,17 +1,78 @@
-/* eslint-disable no-unused-vars */
 import * as React from "react";
+import Layout from "../components/layout";
 import { Link } from "gatsby";
 
-import Layout from "../components/layout";
-import SEO from "../components/seo";
+import { Typography, Row, Col, Card, List } from "antd";
+
+const cardSets = [
+	{
+		title: "For Makers",
+		links: [
+			{
+				url: "/",
+				content: "OSH data submission guideline"
+			},
+			{
+				url: "/",
+				content: "Optimize stored OSH data"
+			}
+		]
+	},
+	{
+		title: "For Developers",
+		links: [
+			{
+				url: "/",
+				content: "Improve the LOSH experience"
+			},
+			{
+				url: "/",
+				content: "Fix bugs in the LOSH app"
+			}
+		]
+	}
+];
 
+interface CardSet {
+	title: string;
+	links: Array<{
+		url: string;
+		content: string;
+	}>;
+}
+
+interface GetInvolvedCardProps {
+	cardSet: CardSet;
```

Comment:
This doesn't match the usage on line 48. Again I think something is up with the Typescript configuration it should have warned you about this.